### ISSUE TITLE
Beaded septum sign

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6687]
**Preferred term label:**
Beaded septum sign

**Synonyms**

**Definition (free text, please give PubMed ID)**
Irregular and nodular thickening of interlobular septa reminiscent of a row of beads (Fig 10). 

PMID:18195376

Comment
The beaded septum sign is frequently seen in lymphangitic spread of cancer and less often in sarcoidosis.
See Figure 10 of Hansell DM, et al., Fleischner Society: glossary of terms for thoracic imaging. Radiology. 2008 Mar;246(3):697-722.

Parent term (use hpo.jax.org/app)
Abnormal pulmonary thoracic imaging finding HP:0031983

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
